awesomplete.js

```diff
@@ -96,6 +96,7 @@ var _ = function (input, o) {
 			}
 
 			if (li && evt.button === 0) {  // Only select on left click
+				evt.preventDefault();
```

Comment:
@LeaVerou I think it's safe to prevent default action for `mousedown` event on LI element here. Maybe you can se any cases where it might hurt?